ui/repetition.reel/repetition.meta

```diff
@@ -62,16 +62,7 @@
         }
     },
     "blueprint_component_reference": {
```

Comment:
Also could be a good opportunity to rename .meta in .mjson

Reply:
Same as above.